src/renderer/views/Channel/Channel.css

```diff
@@ -65,38 +65,50 @@
 
 .channelSearch {
   width: 200px;
+  margin-left: auto;
   align-self: flex-end;
+  flex: 1 1 0%;
 }
 
 .sortSelect {
-  align-self: flex-end;
+  margin-left: auto;
 }
 
 .channelInfoTabs {
   position: relative;
   width: 100%;
-  margin-top: -16px;
-  margin-bottom: -13px;
+  height: 85px;
```

Comment:
Any reason for constant value of height?

Reply:
@vallode